### PR TITLE
DNS Articles Critical Updates - 2 

### PR DESCRIPTION
**Pull request by itsalyse**
Updates the following articles marked `critical` 

Record Editor - https://docs.google.com/document/d/1Q1UkjfgWyqPJss0gad7x437NHqMhv0_BrcraVVCHtHk/edit#heading=h.xxyajaq79zkv

Transferring a domain to another dnsimple account _(by special request)_  - https://docs.google.com/document/d/1QjN_BHw-uH6J_lt-WAaHVwmWGCC5aLBzKrq-YlYY4tk/edit#heading=h.18s9dieyf50e

Regional Records - https://docs.google.com/document/d/1JmphFP_7PSt7wTQ5_uUTAt-FOwz_jb-Li6s3A56orE4/edit

DNS Hosting - https://docs.google.com/document/d/1Yn2GvscT-itghFvjFYnuDnd1Z4yxbmiwRS6OoyQ4K7U/edit#heading=h.rt0cjm1d0otm

Resolution Status - https://docs.google.com/document/d/1-wquy14vmzbBfaoTOCjujY5QuXi39D-ow_tvLIhjgYw/edit#heading=h.88eofxw3gup

Related to https://github.com/dnsimple/dnsimple-business/issues/1900

### CODE REVIEW

File: content/articles/resolution-status.md [hunk 1]
```diff
@@ -7,16 +7,24 @@ categories:
...
+The resolution status shows the name servers currently configured at the registry. Name servers are responsible for answering DNS queries for your domain, and their status is important to your domain functioning properly.
 
-![Domain is resolving](/files/resolution-status-resolving.png)
+## Domains registered with DNSimple
 
-## Domain registered with DNSimple
+If you registered the domain with us, we'll set the name servers automatically to DNSimple. 
 
-If you registered the domain with us, we'll set the name servers automatically to DNSimple. If you updated the name servers to another provider, those will appear as well.
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/resolution-status.md:16]: https://drive.google.com/file/d/1V7Kk80TKjKGKtIL53iEOhBc1XV_-PzSN/view?usp=share_link

File: content/articles/resolution-status.md [hunk 2]
```diff
@@ -7,16 +7,24 @@ categories:
...
 
-If you transferred the domain to us, we don't automatically switch your name servers. [Switch the name servers to us, once you're ready](/articles/delegating-dnsimple-registered/).
+If you updated the name servers to another provider, those will appear as well.
 
-## Domain registered with other registrar
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/resolution-status.md:20]: https://drive.google.com/file/d/1V7Kk80TKjKGKtIL53iEOhBc1XV_-PzSN/view?usp=sharing

File: content/articles/transferring-domain-between-accounts.md [hunk 3]
```diff
@@ -29,32 +31,36 @@ When transferring a domain to another DNSimple account, the domain's registrant
 
 ## Initiating a transfer
 
-To transfer a domain to another DNSimple account, go to the domain's management page, click on the _Settings_ tab and then follow the <label>Transfer ownership</label> link.
+1. Click on the relevant domain. 
+1. Click the Settings tab on the left side.
+1. In the Share or transfer domain card, click Transfer.
+
+    <!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/transferring-domain-between-accounts.md:38]: https://drive.google.com/file/d/1V7Kk80TKjKGKtIL53iEOhBc1XV_-PzSN/view?usp=sharing

File: content/articles/dns-hosting.md [hunk 4]
```diff
@@ -61,37 +60,34 @@ You can activate the DNS hosting service for a domain from the domain DNS tab.
...
+1.  Click the relevant domain name to access the domain page.
+1.  On the left side, click the <label>DNS</label> tab.
 
     ![Domain DNS tab link](/files/domain-tab-dns-link.png)
 
-1.  Find the <label>DNS active zone</label> card, and follow the instructions to activate the zone.
+1.  On the card <label>DNS zone is inactive</label> card, click **Activate zone**.
 
-    ![Enable DNS hosting](/files/domain-dns-hosting-enable.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/dns-hosting.md:71]: https://drive.google.com/file/d/1aAi2a-UyL0KrskwGQSRkKlsJKPLHNr6w/view?usp=sharing

File: content/articles/record-editor.md [hunk 5]
```diff
@@ -18,19 +18,18 @@ The **Record Editor** is the DNSimple DNS editor. The editor gives you the abili
...
-
 ## Locating the record editor
 
-To access the record editor from the domain page, click on the records icon.
-
-![Record Editor from the domain page](/files/rec-editor-domain-page.png)
+1. Click the relevant domain name.
+1. Click the DNS badge in the top right.
 
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/record-editor.md:26]: https://drive.google.com/file/d/17o6ZQjxmiinRWoRFjj29CA7kquKBgKuP/view?usp=sharing

File: content/articles/record-editor.md [hunk 6]
```diff
@@ -18,19 +18,18 @@ The **Record Editor** is the DNSimple DNS editor. The editor gives you the abili
...
 
 ## System records
 
 System Records are special DNS records created by DNSimple when you add a domain to your account. They're necessary to resolve your domain. They can't be edited from the record editor, because they're managed by the DNSimple system.
 
-![System Records](/files/rec-editor-page.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/record-editor.md:32]: https://drive.google.com/file/d/1RDK9YXx8Z-22WiraFdftdMxcajYVlEgH/view?usp=sharing

File: content/articles/dns-hosting.md [hunk 7]
```diff
@@ -61,44 +58,40 @@ You can activate the DNS hosting service for a domain from the domain DNS tab.
...
-1.  Log in to DNSimple with your user credentials.
-1.  If you have multiple accounts, select your account in the navigation bar.
-1.  On the top-nav menu click the <label>Domains</label> tab, locate the relevant domain, and click on the name to access the domain page.
-1.  Switch to the <label>DNS</label> tab.
+1.  If you have multiple accounts, select the relevant account in the navigation bar.
+1.  Click the relevant domain name to access the domain page.
+1.  On the left side, click the <label>DNS</label> tab.
 
-    ![Domain DNS tab link](/files/domain-tab-dns-link.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/dns-hosting.md:65]: https://drive.google.com/file/d/1iFl_gCSq0GEZ3ek8O38qSf8qkBG9Uo13/view?usp=sharing

File: content/articles/dns-hosting.md [hunk 8]
```diff
@@ -61,44 +58,40 @@ You can activate the DNS hosting service for a domain from the domain DNS tab.
...
-1.  Log in to DNSimple with your user credentials.
-1.  If you have multiple accounts, select your account in the navigation bar.
-1.  Click the <label>Domains</label> tab, and locate the relevant domain name to access the domain page.
-1.  Switch to the <label>DNS</label> tab.
+1.  If you have multiple accounts, select the relevant account in the navigation bar.
+1.  Click the relevant domain name to access the domain page.
+1.  On the left side, click the <label>DNS</label> tab.
 
-    ![Domain DNS tab link](/files/domain-tab-dns-link.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/dns-hosting.md:84]: https://drive.google.com/file/d/1iFl_gCSq0GEZ3ek8O38qSf8qkBG9Uo13/view?usp=sharing

File: content/articles/record-editor.md [hunk 9]
```diff
@@ -71,19 +74,23 @@ You can always switch back to the simple editor using the tabs on top. This can
 
 To modify a record, click on the edit icon at the end of the record row.
 
-![Update Record](/files/rec-editor-edit.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/record-editor.md:77]: https://drive.google.com/file/d/13TYPtEQkqMEsKeh6vMjCjqDs_D7KRXCB/view?usp=sharing

File: content/articles/record-editor.md [hunk 10]
```diff
@@ -71,19 +74,23 @@ You can always switch back to the simple editor using the tabs on top. This can
...
 
 To delete a record, click on the trash icon at the end of the record row.
 
 <warning>
-Please note there's no undo. The only way to restore a deleted record is to create it again.
+There is no way to undo deleting a record. The only way to restore a deleted record is to create it again.
 </warning>
 
-![Delete Record](/files/rec-editor-delete.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/record-editor.md:88]: https://drive.google.com/file/d/1VDz1VQNExJ78d_a0q5YWIDS0UKKamG4Q/view?usp=sharing

File: content/articles/record-editor.md [hunk 11]
```diff
@@ -14,49 +14,52 @@ categories:
...
+Enter the required information, and click <label>Add Record</label> to create the record.
 
 ## Record fields editor
 
 Some DNS record types count with a special form that breaks the record content's parts into separate fields. This form provides enhanced error messages to help address any issues.
 
 You can always switch back to the simple editor using the tabs on top. This can be useful when pasting DNS record content from a third-party service provider.
 
-![Simple editor tab](/files/rec-editor-simple-editor-tab.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/record-editor.md:62]: https://drive.google.com/file/d/1qQEcO_0WPpuxMslBbx666_re39TeQhbE/view?usp=sharing

File: content/articles/regional-records.md [hunk 12]
```diff
@@ -8,35 +8,32 @@ categories:
...
 We have <%= POPS.count %> points of presence:
 
 <% POPS.each do |server| %>* <%= server %>
 <% end %>
 
-When creating a new record, select the point of presence for this record. **If you don't select any, the record will appear in all of them.**
+When you're setting up a new record for your domain, you have the option to specify its Points of Presence (POPs). **If you don't specify any, the record will be made available across all DNSimple's Points of Presence by default.**
 
-This feature is enabled by default on certain plans. [See the differences between plans](/articles/dnsimple-plans/).
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/regional-records.md:23]: https://drive.google.com/file/d/1yv3x3aUUaZeRFrfE3-zRx1dSr3l3tRij/view?usp=sharing

File: content/articles/resolution-status.md [hunk 13]
```diff
@@ -7,16 +7,28 @@ categories:
...
+3. You'll see the name servers listed in the **Delegation** card. 
 
-If you transferred the domain to us, we don't automatically switch your name servers. [Switch the name servers to us, once you're ready](/articles/delegating-dnsimple-registered/).
+<!--- needs screenshot -->
 
-## Domain registered with other registrar
+If you updated the name servers to another provider, those will appear as well.
 
-If the domain is registered with another registrar, and you're using DNSimple only for DNS hosting, we currently don't show the name servers for your domain. If you want to point the name servers to us, [please follow these instructions](/articles/delegating-dnsimple-hosted).
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/resolution-status.md:24]: https://drive.google.com/file/d/1uxKBahcnEAw6rMVn_OwCSTM9UNL2kyzV/view?usp=sharing

File: content/articles/dns-hosting.md [hunk 14]
```diff
@@ -31,8 +30,7 @@ If you want to use DNSimple to manage the DNS of an existing domain, you have tw
 
 If you don't have a domain yet, you can [register a domain](https://dnsimple.com/tlds) with us, and subscribe to a DNS hosting plan to manage the domain with us.
```
Comment (Pemacaa) [inline content/articles/dns-hosting.md:31]: What about also adding they can transfer a domain to us?

> If you don't have a domain yet, you can either [register](https://dnsimple.com/tlds)) or [transfer](https://support.dnsimple.com/articles/domain-transfer/) a domain to us and then subscribe to a DNS hosting plan to manage it with us.

File: content/articles/transferring-domain-between-accounts.md [hunk 15]
```diff
@@ -15,46 +15,57 @@ categories:
...
 
-Enter the destination email address of another DNSimple account. Make sure the email address you enter matches the correct account. Once the domain has been transferred, you won't be able to control this domain under your current account.
+1. Enter the destination email address of another DNSimple account.
+1. Click **Move domain**.
+1. A banner will appear confirming the request to push the domain. 
 
 <warning>
-Make sure to use the [account email address](/articles/changing-account-email/), rather than a [user email address](/articles/changing-user-email/). Otherwise, the push will not be successful.
+Make sure to correctly enter the account email address. Once the domain has been transferred, you won't be able to control it under your current account.
```
Comment (Pemacaa) [inline content/articles/transferring-domain-between-accounts.md:45]: I would rewrite this to make it clear that when they start the transfer and if they write the email wrong, they won't be able to cancel the transfer on their end.

> Be sure to enter the correct account email address, as you won't be able to cancel the transfer on your own if the email is incorrect. If the email is wrong and you need to cancel the transfer, you will have to contact support for assistance.

**Review by Pemacaa — APPROVED** [review]

File: content/articles/transferring-domain-between-accounts.md (hunk 15 above)
Comment (stephanoodle) [reply content/articles/transferring-domain-between-accounts.md:45]: I agree with Carla that we should add this.

**Review by stephanoodle — APPROVED** [review]

File: content/articles/dns-hosting.md (hunk 14 above)
Comment (itsalyse) [reply content/articles/dns-hosting.md:31]: Who can say? That's in the existing article. We can just say 'plan' and link the pricing page

**Comment by itsalyse** [comment]
@Pemacaa @stephanoodle any objections to changing that section in the transfer article to this? The warning was getting really long and I think there's an opportunity to break it up a bit 
Screenshot 2024-09-24 at 2 41 30 PM

File: content/articles/dns-hosting.md (hunk 14 above)
Comment (Pemacaa) [reply content/articles/dns-hosting.md:31]: I thought about it, and you're right it t's not necessary. Plus, we already covered it in the second point.

Comment (itsalyse) [reply content/articles/dns-hosting.md:31]: Hang on, don't you have to be subscribed to a plan to register a domain in the first place? Would it make more sense to say 
"If you don't have a domain yet, you can [sign up](plan page link) to [register a domain](https://dnsimple.com/tlds) and activate the zone to manage your DNS with us." ?

Comment (Pemacaa) [reply content/articles/dns-hosting.md:31]: You are totally correct, and the way you formulated the sentences makes more sense. Maybe add "or log in" as well? 

> If you don't have a domain yet, you can [sign up](plan page link) or log in to [register a domain](https://dnsimple.com/tlds) and activate the zone to manage your DNS with us.

**Comment by Pemacaa** [comment]
> @Pemacaa @stephanoodle any objections to changing that section in the transfer article to this? The warning was getting really long and I think there's an opportunity to break it up a bit Screenshot 2024-09-24 at 2 41 30 PM

No objections, it looks good to me!

**Comment by stephanoodle** [comment]
> @Pemacaa @stephanoodle any objections to changing that section in the transfer article to this? 

No objection here either.

**Comment by itsalyse** [comment]
@dperrymorrow the content has been finalized — this is ready for screenshots to be added

**Comment by itsalyse** [comment]
Hey @dperrymorrow could you add the screenshots so we can get this one merged? thanks!

File: content/articles/dns-hosting.md [hunk 16]
```diff
@@ -61,49 +58,49 @@ You can activate the DNS hosting service for a domain from the domain DNS tab.
...
-1.  Switch to the <label>DNS</label> tab.
+1.  If you have multiple accounts, select the relevant account in the navigation bar.
+1.  Click the relevant domain name to access the domain page.
+1.  On the left side, click the <label>DNS</label> tab.
 
-    ![Domain DNS tab link](/files/domain-tab-dns-link.png)
+<!--- needs screenshot -->
 
-1.  Find the <label>DNS active zone</label> card, and follow the instructions to activate the zone.
+1.  On the <label>DNS zone is inactive</label> card, click **Activate zone**.
```
Comment (itsalyse) [inline content/articles/dns-hosting.md:67]: check numbering after adding screenshot

File: content/articles/dns-hosting.md [hunk 17]
```diff
@@ -61,49 +58,49 @@ You can activate the DNS hosting service for a domain from the domain DNS tab.
...
-1.  Switch to the <label>DNS</label> tab.
+1.  If you have multiple accounts, select the relevant account in the navigation bar.
+1.  Click the relevant domain name to access the domain page.
+1.  On the left side, click the <label>DNS</label> tab.
 
-    ![Domain DNS tab link](/files/domain-tab-dns-link.png)
+<!--- needs screenshot -->
 
-1.  Find the <label>DNS Hosting</label> card and follow the instructions to disable DNS hosting.
+1.  On the <label>DNS zone is active</label> card, click **Deactivate zone**.
```
Comment (itsalyse) [inline content/articles/dns-hosting.md:86]: check numbering after adding screenshot

**Comment by dperrymorrow** [comment]
@itsalyse the screenshots have been updated, please review the preview and Ill deploy

**Pull request merged**